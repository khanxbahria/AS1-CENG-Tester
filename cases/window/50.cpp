#include "../../your_code/Window.cpp"
#include "../../your_code/Browser.cpp"
#include "../../your_code/LinkedList.h"
int main() {
{
Window* wind = new Window();
wind->isEmpty();
Tab G = Tab("qqq", "vv","jjj");
wind->print();
Tab S = Tab("s", "t","g");
wind->print();
Tab M = Tab("zzz", "cc","qq");
wind->closeTab();
Tab W = Tab("fff", "nnn","aa");
wind->moveActiveTabTo(5);
Tab K = Tab("bb", "iii","ccc");
wind->changeActiveTabTo(2);
Tab Y = Tab("k", "aaa","c");
wind->print();
Tab C = Tab("hh", "y","vvv");
wind->moveActiveTabTo(3);
wind->isEmpty();
Tab E = Tab("ll", "fff","oo");
wind->closeTab();
wind->getActiveTab();
Tab Q = Tab("y", "uuu","i");
wind->moveActiveTabTo(0);
Tab N = Tab("ggg", "www","uu");
wind->isEmpty();
Tab Z = Tab("c", "mmm","rr");
wind->newTab(C);
Tab T = Tab("rrr", "mm","ff");
wind->changeActiveTabTo(2);
Tab V = Tab("bb", "w","cc");
wind->moveActiveTabTo(1);
wind->moveActiveTabTo(1);
wind->changeActiveTabTo(4);
Tab I = Tab("m", "a","q");
wind->closeTab();
Tab L = Tab("o", "uuu","tt");
wind->print();
Tab F = Tab("ee", "e","nn");
wind->getActiveTab();
wind->isEmpty();
Tab D = Tab("mm", "f","l");
wind->print();
Tab X = Tab("ll", "qq","aa");
wind->print();
wind->print();
wind->getActiveTab();
wind->getActiveTab();
wind->changeActiveTabTo(0);
Tab J = Tab("aaa", "ff","dd");
wind->print();
wind->print();
wind->moveActiveTabTo(5);
wind->closeTab();
wind->getActiveTab();
wind->isEmpty();
wind->isEmpty();
wind->getActiveTab();
Tab P = Tab("qqq", "dd","ooo");
wind->isEmpty();
wind->getActiveTab();
wind->print();
Tab B = Tab("ddd", "tt","p");
wind->closeTab();
wind->getActiveTab();
wind->moveActiveTabTo(3);
wind->getActiveTab();
wind->newTab(B);
Tab R = Tab("kkk", "ooo","fff");
wind->newTab(D);
Tab A = Tab("fff", "d","f");
wind->getActiveTab();
Tab U = Tab("a", "ee","rr");
wind->changeActiveTabTo(4);
wind->print();
wind->isEmpty();
wind->print();
wind->getActiveTab();
wind->changeActiveTabTo(4);
wind->print();
wind->getActiveTab();
wind->moveActiveTabTo(2);
wind->isEmpty();
wind->print();
wind->changeActiveTabTo(3);
wind->changeActiveTabTo(1);
Tab O = Tab("zz", "mmm","nnn");
wind->newTab(A);
wind->changeActiveTabTo(4);
wind->print();
wind->moveActiveTabTo(2);
wind->changeActiveTabTo(3);
wind->newTab(E);
wind->closeTab();
wind->print();
wind->moveActiveTabTo(0);
wind->getActiveTab();
wind->getActiveTab();
wind->newTab(F);
wind->newTab(G);
wind->closeTab();
wind->getActiveTab();
wind->closeTab();
wind->moveActiveTabTo(0);
wind->closeTab();
wind->closeTab();
wind->print();
wind->print();
wind->isEmpty();
wind->closeTab();
wind->closeTab();
wind->isEmpty();
wind->closeTab();
wind->closeTab();
wind->changeActiveTabTo(5);
wind->getActiveTab();
wind->isEmpty();
wind->closeTab();
wind->closeTab();
wind->closeTab();
wind->print();
wind->moveActiveTabTo(1);
wind->getActiveTab();
wind->isEmpty();
wind->moveActiveTabTo(3);
wind->getActiveTab();
wind->print();
wind->newTab(I);
delete wind;
}
	return 0;
}