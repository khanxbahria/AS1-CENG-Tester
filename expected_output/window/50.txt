The active tab is tab 0




The active tab is tab 0




Window is empty
The list is empty.
Window is empty
The list is empty.
Window is empty
The list is empty.
Window is empty
The list is empty.
Window is empty
The list is empty.
Window is empty
The list is empty.
Window is empty
The list is empty.
Window is empty
The list is empty.
The active tab is tab 1
tt
ddd
p

f
mm
l

The active tab is tab 1
tt
ddd
p

f
mm
l

The active tab is tab 1
tt
ddd
p

f
mm
l

The active tab is tab 1
tt
ddd
p

f
mm
l

The active tab is tab 2
tt
ddd
p

f
mm
l

d
fff
f

The active tab is tab 2
tt
ddd
p

f
mm
l

d
fff
f

The active tab is tab 0
e
ee
nn

The active tab is tab 0
e
ee
nn

Window is empty
The list is empty.
Window is empty
The list is empty.
